local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -106,14 +106,17 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 
 // GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
 // fullPath is the pathname of any file within the mounted filesystem. Capacity
-// returned here is total capacity available to non-root users, and does not include
-// fs reserved space.
+// returned here is total capacity available.
 func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
-	var s unix.Statfs_t
-	if err := unix.Statfs(fullPath, &s); err != nil {
+	//largely copied from kubernetes/pkg/volume/util/fs.go:FsInfo(path)
```

Comment:
@wongma7 - thanks for the advice. I was able to execute the commands.

PTAL and let me know if the additional files in this commit look good.